### PR TITLE
Add compatibility Samples

### PR DESCRIPTION
**Pull request by Youri970410**

### CODE REVIEW

File: Samples/Package/Functional/Calculator-Appium-Python-unittest-Compatibility-Sample/Package/scripts/functional/install-app.ps1 [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+if ($installer.exitcode -eq 0) {
+    Log("Installation successful as $($installer.exitcode)")
+}
+else {
+    Log("Error: Installation failed as $($installer.exitcode)")
+    $exit_code = $installer.exitcode
+}
+Log("Installation finished as $exit_code")
+Add-MpPreference -ExclusionPath  "C:\Program Files (x86)\Calculator\"
+exit $exit_code
```
Comment (yazeng) [inline Samples/Package/Functional/Calculator-Appium-Python-unittest-Compatibility-Sample/Package/scripts/functional/install-app.ps1:53]: add a new line for all needed files.

File: Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+### Compatibility test cases
+It is used to test compatibility with Windows. For example:
+- Create new desktop and launch it from different desktop
+- Search App by Windows Search
+- Max/Min your app
+- Pin/Unpin your app to Start/TaskBar
+- Open/Close your app from TaskBar
+- Open app Settings
+- Terminate and Reset app
+
```
Comment (yazeng) [inline Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md:14]: remove one line

File: Samples/Package/Functional/Calculator-Appium-Python-unittest-Compatibility-Sample/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### Compatibility test cases
+It is used to test your app's compatibility with Windows. For example:
+- Create new desktop and launch it from different desktop
+- Search App by Windows Search
+- Max/Min your app
+- Pin/Unpin your app to Start/TaskBar
+- Open/Close your app from TaskBar
+
+### ScreentShot
```
Comment (yazeng) [inline Samples/Package/Functional/Calculator-Appium-Python-unittest-Compatibility-Sample/README.md:23]: Take Screenshots

File: Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+This package sample includes one functional test Script: [TodoTest.py](./Tests). TodoTest.py is a python functional test script based on the '**unittest**' Test Framework to run and validate basic UI scenarios on **Microsoft To Do** application. It highlights the following basic interactions to demonstrate how UI testing using [Windows Application Driver](https://github.com/Microsoft/WinAppDriver).
+- Creating a modern UWP app session
+- Finding element using name
+- Finding element using XPath
+- Sending click action to an element
+- Retrieving element value
+
+*A Functional test executes your uploaded test script(s) on your package. The scripts are run in the sequence you specified and a failure in a particular script will stop subsequent scripts from executing.*
+
+### Take ScreentShot
```
Comment (yazeng) [inline Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md:26]: Take Screenshots

File: Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+# Microsoft To Do Compatibility Test Package (Appium Python unittest Test Cases)
+## Microsoft To do
```
Comment (jujiang-msft) [inline Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md:2]: `To do` vs. `To Do` let's keep consistent

File: Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+This package sample is using Microsoft To Do as the test target. Microsoft To Do is an App in Microsoft Store. Click [here](https://apps.microsoft.com/store/detail/microsoft-to-do-lists-tasks-reminders/9NBLGGH5R558) for details of Microsoft To Do.
```
Comment (jujiang-msft) [inline Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/README.md:3]: Also keep consistent if 'Microsoft To Do' needs to be bold

File: Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/Tests/TodoTest.py [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+            self.driver.find_element_by_name("Microsoft To Do, App, Press right to switch preview").click()
+            sleep(5)
+
+if __name__ == '__main__':
+
+    unittest.main(
+        testRunner=xmlrunner.XMLTestRunner(output='test-reports'),
+        failfast=False,
+        buffer=False,
+        catchbreak=False)
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply Samples/Package/Functional/Todo-Appium-Python-unittest-Compatibility-Sample/Tests/TodoTest.py:165]: added

**Review by yazeng — APPROVED** [review]

**Pull request merged**